src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylePairs = sourceString
+    .trim()
+    .split(';')
+    .map((pair) => pair.trim().split(':'))
+    .filter((pair) => pair.length > 1);
```

Comment:
The chained methods should have the same indentation to improve readability. Align the dot operators vertically.